defender/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
```

Comment:
We could remove all the carets `^` to keep it consistent with the update that `AE-1474` made

Reply:
Good idea, removed.